test/test_browser.py

```diff
@@ -2643,6 +2643,7 @@ def test_html5_webgl_create_context2(self):
     self.btest_exit('webgl_create_context2.c')
 
   @requires_graphics_hardware
+  @requires_offscreen_canvas
```

Comment:
Doesn't only one of the variants require it?

Reply:
Yeah, that's right.

I thought to keep this change simple and over-conservative, to avoid too much churn from testing old browsers.